app/controllers/concerns/exception_handler.rb

```diff
@@ -33,11 +34,12 @@ def render_record_not_found(exception)
 
   def render_record_invalid(exception)
     errors = exception.record.errors.messages
+
     render_attributes_errors(errors)
   end
 
   def render_page_not_found
-    render_errors(I18n.t('errors.page_not_found'), :bad_request)
+    render_errors(I18n.t('errors.page_not_found'), :unprocessable_entity)
```

Comment:
Shouldn't this be not_found?

Reply:
This method is rescuing from [`Pagy::OverflowError`](https://ddnexus.github.io/pagy/extras/overflow) do you still think that is a `not_found`?

Reply:
mmm, I will check out what is the norm there, not sure

Reply:
@mwolman Just tried it in GitHub pagination and not_found is not a good fit. unprocessable_entity makes more sense here 🙌

Reply:
However, I would change the function name to something like render_page_overflow, I totally thought it was a page not found response 😕 . What do you think? maybe someone else has an opinion here too

Reply:
I agree with you 100%, the method name is not descriptive. I'll change it